packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -597,6 +611,60 @@ export function getBeaconBlockApi({
         },
       };
     },
+
+    async getBlobs({blockId, indices}) {
+      assertUniqueItems(indices, "Duplicate indices provided");
+
+      const {block, executionOptimistic, finalized} = await getBlockResponse(chain, blockId);
+      const fork = config.getForkName(block.message.slot);
+      const blockRoot = sszTypesFor(fork).BeaconBlock.hashTreeRoot(block.message);
+
+      let blobs: deneb.Blobs;
+
+      if (isForkPostFulu(fork)) {
+        if (chain.custodyConfig.targetCustodyGroupCount !== NUMBER_OF_COLUMNS) {
+          throw Error(`Must custody all ${NUMBER_OF_COLUMNS} data columns to be able to serve blobs`);
```

Comment:
could consider to allow usage of this api if >64 columns are custodied and reconstruct full data matrix via `recoverDataColumnSidecars`

Reply:
can you also include `targetCustodyGroupCount` to the error message
also either recover data columns now, or leave a `TODO` comment for it